utils/helper.js

```diff
@@ -22,7 +23,14 @@ const amendHandleBarTemplate = (
   });
   const payload = `${issueModule}_${v4()}_payload.json`;
 
-  fs.writeFileSync(`${config.UTILS.PAYLOADS_DIR}/${payload}`, templateModifier, 'utf8');
+  let beautifiedTemplate;
+  try {
+    beautifiedTemplate = JSON.stringify(dirtyJSON.parse(templateModifier));
+  } catch (e) {
+    console.error(`Vulnerability "${issueSummary}" cannot be turned into a Jira issue since the data provided in the given JSON are malformed and cannot be beautified!`);
+  }
+
```

Comment:
In case of an exception, please consider to re-throw, catch in the caller and fail the action with `core.setFailed(e)` if not intended to handle otherwise.